spec/OAuth1.spec.js

```diff
@@ -87,25 +87,33 @@ describe('OAuth', function () {
     done();
   }
 
-  it('Should fail a GET request', done => {
+  it('GET request should fail with invalid credentials', done => {
```

Comment:
Looks already much better. Can you just add `for a resource that requires OAuth` to both tests? To explicitly describe the purpose of the test for future changes.